src/modeler/AutoRest.Swagger/OperationBuilder.cs

```diff
@@ -148,9 +148,15 @@ public Method BuildMethod(HttpMethod httpMethod, string url, string methodName,
                 headerType = null;
             }
 
+            
             // Response format
             List<Stack<IModelType>> typesList = BuildResponses(method, headerType);
 
+            method.IsXNullableReturnType = (_operation.Responses.Keys.All(key =>
```

Comment:
generally: does `IsXNullableReturnType` have to be set here? This is very early in the pipeline so response types *might* be changed by later steps and extensions! If possible: make `IsXNullableReturnType` a derived property that simply does precisely what you do here, but precisely at the time you query for it instead of ahead of time.